### ISSUE TITLE
x/build: decide status of Android TryBots

### ISSUE THREAD

**Issue by bcmills** [golang/go#53377]
The `android-386-emu` and `android-amd64-emu` builders currently run as TryBots in the default set. However, `android/386` and `android/amd64` are not [first-class ports](https://go.dev/wiki/PortingPolicy#first-class-ports), and have some open issues that can cause test failures in TryBot runs (notably #51001, #52724, #42212).

Discussion #53060 proposes to run only first-class ports as TryBots, but I think we should on the status of `android` TryBots in particular independent of that discussion, because we have specific known failure modes affecting the `android` TryBots today.

[CL 407615](https://go.dev/cl/407615) would remove these TryBots, and #25963 discusses `misc-compile` TryBots that could plausibly take their place.

In order to avoid TryBot failure noise when the tree reopens for Go 1.20, we should decide on the status of these TryBots before completing the Go 1.19 release cycle.

(CC @eliben @golang/android)

**Comment by dmitshur** [comment]
As a small correction, only the `android-amd64-emu` builder is configured as a TryBot. The `android-386-emu` one is a post-submit only builder for all repos.

**Comment by dmitshur** [comment]
I think in the current state it makes sense for the `android-amd64-emu` builder to be post-submit only.

If I understand correctly, @cherrymui may support that too, based on [this comment](https://go-review.googlesource.com/c/build/+/407615/comments/f28b3484_64c1e536) on CL 407615. (I'll update that CL once this issue moves to NeedsFix state.)

**Comment by changkun** [comment]
Just clarify questions: 
What's the exact consequence of removing a builder from TryBots? 
Does it mean that this port will never be tested unless explicitly requested? 
Is it going to be similar to how iOS living today? (i.e. unless `try=ios`, the builder will only be tested after a CL is submitted?) 
Instead of removing it from TryBots will at least require doing a proper upgrade, is that correct?

**Comment by dmitshur** [comment]
> Is it going to be similar to how iOS living today? (i.e. unless `try=ios`, the builder will only be tested after a CL is submitted?)

Yes, this is correct. "Removing the builder from the TryBot set" would mean it will not run automatically before CL submission when Run-TryBot+1 vote is set, but can be requested via [SlowBots](https://go.dev/wiki/SlowBots) syntax (i.e., `TRY=android` or `TRY=android-amd64`). But it'll run after a CL is submitted (aka post-submit builds, as visible on the [build dashboard](https://build.golang.org/)), that's not changing here.

I think this answers other questions, but let me know if I missed something. (I didn't quite understand the very last question.)
